listen_to_mqtt.sh

```diff
@@ -66,126 +71,157 @@ listen_to_mqtt() {
           ;;
         honk)
-          log_notice "Honk Horn"
-          send_command $vin $msg
+          send_command $vin $msg "Honk horn"
+          ;;
+        list-keys)
+          send_command $vin $msg "List public keys enrolled on vehicle"
           ;;
         lock)
-          log_notice "Lock Car"
-          send_command $vin $msg
+          send_command $vin $msg "Lock vehicle"
+          ;;
+        media-toggle-playback)
+          send_command $vin $msg "Toggle between play/pause"
+          ;;
+        ping)
+          send_command $vin $msg "Ping vehicle"
+          ;;
+        software-update-cancel)
+          send_command $vin $msg "Cancel a pending software update"
+          ;;
+        software-update-start)
+          send_command $vin $msg "Start software update after delay"
+          ;;
+        tonneau-close)
+          send_command $vin $msg "Close Cybertruck tonneau"
+          ;;
+        tonneau-open)
+          send_command $vin $msg "Open Cybertruck tonneau"
+          ;;
+        tonneau-stop)
+          send_command $vin $msg "Stop moving Cybertruck tonneau"
+          ;;
+        trunk-close)
+          send_command $vin $msg "Close vehicle trunk"
+          ;;
+        trunk-move)
+          send_command $vin $msg "Toggle trunk open/closed"
+          ;;
+        trunk-open)
+          send_command $vin $msg "Open vehicle trunk"
           ;;
         unlock)
-          log_notice "Unlock Car"
-          send_command $vin $msg
+          send_command $vin $msg "Unlock vehicle"
+          ;;
+        wake)
+          send_command $vin " "-domain vcsec $msg" "Wake up vehicule"
           ;;
         windows-close)
-          log_notice "Close Windows"
-          send_command $vin $msg
+          send_command $vin $msg "Close all windows"
           ;;
         windows-vent)
-          log_notice "Vent Windows"
-          send_command $vin $msg
+          send_command $vin $msg "Vent all windows"
           ;;
         *)
-          log_error "Invalid command request; topic:$topic msg:$msg"
+          log_error "Invalid command request; vin:$vin topic:$topic msg:$msg"
           ;;
         esac
         ;; ## END of command)
 
+      auto-seat-and-climate)
+        send_command $vin "auto-seat-and-climate LR on" "Turn on automatic seat heating and HVAC"
+        ;;
+
+      charging-schedule)
+        send_command $vin "charging-schedule $msg" "Schedule charging to $msg minutes after midnight and enable daily scheduling"
+        ;;
+
       charging-set-amps)
         # https://github.com/iainbullock/tesla_ble_mqtt_docker/issues/4
         if [ $msg -gt 4 ]; then
-          log_notice "Set amps"
-          send_command $vin "charging-set-amps $msg"
+          send_command $vin "charging-set-amps $msg" "Set amps"
         else
-          log_notice "First Amp set"
-          send_command $vin "charging-set-amps $msg"
+          send_command $vin "charging-set-amps $msg" "First Amp set"
```

Comment:
cmd_name: "Set charging Amps to 5A then to $msg"

Reply:
Done.